CHANGES/10099.bugfix.rst

```diff
@@ -0,0 +1 @@
+Fixed unknown `SSLContext` symbol for type checkers
```

Comment:
```suggestion
Fixed unknown `SSLContext` symbol for type checkers -- by :user:`AlanBogarin`.
```

Reply:
It looks like this suggestion was marked as resolved without any context or commentary.